README.md

```diff
@@ -36,7 +36,7 @@ application dependency:
 
 ```elixir
 def application do
-  [applications: [:httpoison]]
+  [extra_applications: [:httpoison]]
```

Comment:
I disagree that this should be explicit. The new feature that shipped with Elixir 1.4 is application **inference**, which implies that you no longer need to be explicit. 

From [v1.4 Release Notes](https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/):

"Mix will automatically build your application list based on your dependencies. Developers now only need to specify which applications shipped as part of Erlang or Elixir that they require, such as :logger."

I would just make a note to only add `:httpoison` to your application list if your elixir version is 1.3 or lower.